clients/js/src/index.ts

```diff
@@ -29,7 +29,7 @@ export class OpenAIEmbeddingFunction {
   private org_id: string;
   private model: string;
 
-  constructor(openai_api_key: string, openai_model?: string, openai_organization_id?: string) {
+  constructor(api_key: string, model?: string, org_id?: string) {
```

Comment:
sorry can you roll the `OpenAI` stuff back so we don't incur a breaking change here?

Reply:
rolled back.